### DISCUSSION TITLE
Elasticsearch source

### DISCUSSION THREAD

**Discussion by sundara20** [vectordotdev/vector#11270] [Q&A] [answered]
Does Vector have an Elasticsearch source option like the Logstash Elastic input plugin (https://www.elastic.co/guide/en/logstash/current/plugins-inputs-elasticsearch.html)? If not, is there a plan to support this?

**Comment by spencergilbert** [comment, marked as answer]
Hi @sundara20 - there is currently no source for `elasticsearch`, if you'd like to open a feature request issue with your usecase and to gauge user support that would be great!

**Reply by StephenWakely** [reply]
We do have an issue #10170 where this has been requested.

**Reply by spencergilbert** [reply]
Ah - of course searching for `elasticsearch source` didn't pull that up 🙃

**Reply by sundara20** [reply]
@spencergilbert Yup 😂
